csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -50,6 +50,22 @@ public bool SetSessionGraphOptimizationLevel(uint optimization_level)
             return result == 0;
         }
 
+        /// <summary>
+        /// Enable parallel execution.
+        /// </summary>
+        /// <param name="enable">
+        /// Call with true to enable Parallel Execution
+        /// Call with false to disable Parallel Execution
+        /// By default, Parallel Execution is disabled
+        /// </param>
+        public void EnableParallelExecution(bool enable=true)
+        {
+            if (enable)
+                NativeMethods.OrtDisableSequentialExecution(_nativePtr);
+            else
+                NativeMethods.OrtEnableSequentialExecution(_nativePtr);
+        }
```

Comment:
Refactor EnableParallelExecution() into two separate functions, to mirror the C-API more closely.

So "DisableSequentialExecution()" and "EnableSequentialExecution()"